application/src/main/java/org/togetherjava/tjbot/config/FreeCommandConfig.java

```diff
@@ -0,0 +1,60 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Config instance for the Free Command System see
+ * {@link org.togetherjava.tjbot.commands.free.FreeCommand}
+ *
+ * The Json looks as follows:
+ * 
+ * <pre>
+ * "freeCommand": [
+ *   {
+ *       "statusChannel": long_number,
+ *       "monitoredChannels": [long_number, long_number]
+ *   }]
+ * </pre>
+ * 
+ * Additional Guilds may add their settings by adding additional {"statusChannel....... }
+ *
+ * The long channel ID can be found by right-clicking on the channel and selecting 'Copy ID'
+ */
+@SuppressWarnings("ClassCanBeRecord")
```

Comment:
Why not make it a record?

Reply:
1) Consistency with its (what's the opposite of composition?) parent

2) The constructor is private, which is not allowed for records

3) Records are too limiting, I have never made something a record that I later had to change back when I add more functionality. The only possible exception is DTO's since they literally dont require any behaviour.